<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preco</class>
 <widget class="QDialog" name="Preco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Preço de Item</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1041</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>Gerenciar preço de itens</string>
   </property>
   <widget class="QTableView" name="tableView_itens">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>451</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_filtraritens">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filtraritens">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Preço</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_apagar">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Apagar
Preço</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_filtrarpreco">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>120</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_tamanhoitem">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_item">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>[ Item selecionado ]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_preco">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>000.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filtrarpreco">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>441</width>
      <height>22</height>
     </rect>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_tipoitem">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_inserir">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserir</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_precos">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>531</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
